packages/editor/src/components/ComponentForm/JsonSchemaForm/TopLevelField.tsx

```diff
@@ -18,6 +18,23 @@ type ExpandSpec = {
   name?: string;
 };
 
+const PRESET_PROPERTY_CATEGORY_WEIGHT = {
+  Data: 100,
+  Basic: 99,
+  Behavior: 3,
+  Layout: 2,
+  Style: 1,
+  Advance: -Infinity,
+};
+
+function getCategoryOrder(name: string): number {
+  return name in PRESET_PROPERTY_CATEGORY_WEIGHT
```

Comment:
Why not just `PRESET_PROPERTY_CATEGORY_WEIGHT[name] || 0`?

Reply:
I copied this function from [ComponentList](https://github.com/webzard-io/sunmao-ui/blob/main/packages/editor/src/components/ComponentsList/ComponentList.tsx)，I think both expressions are fine